src/main/java/com/books/controllers/BookController.java

```diff
@@ -33,12 +33,15 @@ public BookController(BookService bookService) {
      * @return all books / all books with the given string contained in the title name (if param is provided).
      */
     @GetMapping(path = "/books", produces = "application/json")
-    public ResponseEntity<List<Book>> getAllBooks(@RequestParam(required = false) String titleContaining){
+    public ResponseEntity<List<Book>> getAllBooks(
```

Comment:
Rename to just getBooks because the response is never all available books, but a page of filtered by title or not

Reply:
Done